source/deprecations/v3.x.html.md

```diff
@@ -53,3 +53,42 @@ object.notifyPropertyChange('someProperty');
 ##### id: ember-metal.getting-each
 
 Calling `array.get('@each')` is deprecated. `@each` may only be used as dependency key.
+
+#### Use console rather than Ember.Logger
+
+##### until: 3.5.0
+##### id: ember-console.deprecate-logger
+
+The `Ember.Logger` module will be removed after the first LTS of the 3.x cycle. You should remove any calls to `Ember.Logger` and use calls to `console` instead.
+
+In Edge and IE11, uses of `console` beyond calling its methods may require more subtle changes than simply substituting `console` wherever `Logger` appears. In these browsers, they will behave just as they do in other browsers when your development tools window is open. However, when run normally, calls to its methods must not be bound to anything other than the console object or you will receive an `Invalid calling object` exception. This is a known inconsistency with these browsers. (Edge issue #14495220.)
+
+For example, the following uses will not work in Edge and IE11 except when the development tools window is open:
+
+``` javascript
+var print = console.log; // print is unbound, therefore bound to global
```

Comment:
Why is this mentioned? Seems unrelated to the deprecation.

Reply:
When the user changes their code to silence the deprecation, it will at first appear that a simple text substitution of “Logger”=>”console” will do. However, the moment anybody runs their code with Edge or IE11 *without the debugger open*, they’ll face-plant into this issue. The solution is simple but non-obvious, so I figured I’d better document it so anybody who is updating their code by hand will be equipped to handle it as they make their change. This is something the codemod will also need to deal with or flag for manual intervention.